doc/sphinx-guides/source/admin/dataverses-datasets.rst

```diff
@@ -147,7 +147,7 @@ It returns a list in the following format:
 Unlink a Dataset
 ^^^^^^^^^^^^^^^^
 
-Removes a link between a dataset and a Dataverse collection. Only accessible to superusers. ::
+Removes a link between a dataset and a Dataverse collection. Accessible to superusers or Users with Publish Dataset permissions (as of Dataverse 6.4). ::
```

Comment:
```suggestion
Removes a link between a dataset and a Dataverse collection. Accessible to users with Publish Dataset permissions. ::
```

No need to mention superusers anymore, I'd say. They bypass the permission system.

Also, I think it's ok to not mention 6.4. The info is in the release note.